src/api/app/controllers/webui/staging/workflows_controller.rb

```diff
@@ -72,16 +72,7 @@ def update
 
     begin
       Staging::Workflow.transaction do
-        old_managers_group = @staging_workflow.managers_group
-        new_managers_group = Group.find_by(title: params[:managers_title])
-
-        @staging_workflow.staging_projects.each do |staging_project|
-          staging_project.unassign_managers_group(old_managers_group)
-          staging_project.assign_managers_group(new_managers_group)
-          staging_project.store
-        end
-
-        @staging_workflow.managers_group = new_managers_group
+        @staging_workflow.managers_group = Group.find_by(title: params[:managers_title])
```

Comment:
Why does it still need to be in a transaction and in a rescue? Why not doing it standard way?

```ruby
  @staging_workflow.managers_group = Group.find_by(title: params[:managers_title])
  if save
    ...
  else
    ...
  end
```